README.md

```diff
@@ -1,18 +1,89 @@
 # The Foreman Release Engineering scripts
 
-These are the scripts I use while releasing [The Foreman](https://theforeman.org). They automate most of the [release process](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb).
+A collection of small scripts, adhering to the [Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy), used to release [The Foreman](https://theforeman.org). They automate parts of the [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb) processes. It is also used for the [Katello](https://theforeman.org/plugins/katello/) [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/release.md.erb).
```

Comment:
```suggestion
A collection of small scripts, adhering to the [Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy), used to release [The Foreman](https://theforeman.org). They automate parts of the [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb) processes. It is also used for the [Katello](https://theforeman.org/plugins/katello/) [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/release.md.erb) processes.
```

(one sentence per line makes the diff nicer to read ;-))

Reply:
Will change.